src/main/webapp/script.js

```diff
@@ -173,6 +173,42 @@ function getGraphDisplay(graphNodes, graphEdges) {
     const node = evt.target;
     node.tip.show();
   });
+
+  let searchElement = document.getElementById('search');
+  searchElement.onchange = function() {
+    if (searchNode(cy, searchElement.value)) {
+      document.getElementById('searcherror').innerText = "";
```

Comment:
if searcherror is empty by default, why is this necessary?

Reply:
I have this so if someone enters an invalid node, then a valid one, the error message will go away.

Reply:
maybe you could consider adding a button rather than doing it onChange. You probably don't need to update the graph every time the box changes (since typing out a node name could take a while?)